src/azul/plugins/repository/tdr_hca/__init__.py

```diff
@@ -189,6 +189,10 @@ def add_entity(self,
             self.stitched.add(entity.entity_id)
         if entity.entity_type.endswith('_file'):
             descriptor = json.loads(row['descriptor'])
+            # FIXME Move validation of descriptor to the metadata API
+            #       https://github.com/DataBiosphere/azul/issues/6299
+            from humancellatlas.data.metadata.api import Entity  # noqa AZL131
```

Comment:
Why is `noqa` used here instead of just properly formatting the import statement?